src/display_manager.py

```
@@ -405,6 +406,19 @@ def _load_fonts(self):
             logger.info(f"Assigned calendar_font (type: {type(self.bdf_5x7_font).__name__}) to bdf_5x7_font.")
 
+            # Load 4x6 BDF font for compact body text (of-the-day descriptions, etc.)
+            try:
+                bdf_4x6_path = "assets/fonts/4x6.bdf"
+                if not os.path.exists(bdf_4x6_path):
+                    raise FileNotFoundError(f"Font file not found at {bdf_4x6_path}")
+                face_4x6 = freetype.Face(bdf_4x6_path)
+                face_4x6.set_pixel_sizes(0, 6)
+                self.bdf_4x6_font = face_4x6
+                logger.info("4x6 BDF font loaded successfully")
+            except Exception as font_err:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)